packages/admin-ui-extensions/src/components/Form/Form.ts

```diff
@@ -0,0 +1,34 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+export interface FormProps {
+  /**
+   * Whether the form is able to be submitted. When set to `true`, this will
+   * disable the implicit submit behavior of the form.
+   */
+  disabled?: boolean;
+  /**
+   * Whether this form should be submitted when the buyer presses the "enter"
+   * key on their keyboard. Defaults to `true`. When set to a string value,
+   * the implicit submit behavior will be enabled, and the string will be used
+   * as a label for an explicit submit button that is visible only to assistive
+   * technologies.
+   */
+  implicitSubmit?: boolean | string;
```

Comment:
@iainmcampbell  I think `disabled` and `implicitSubmit` are non-overlapping in concern. `disable` corresponds to disabling of the form. That could mean if you have a custom `Button.submit` and press it, but it won't call `onSubmit`. You could also have `implicitSubmit=true` and `disabled=true` for a time when the form can't submit.

I should mention the `Button.submit` though.

@olavoasantos I think you could split it if you want, but I prefer checkout's implementation because it's concise. You'd probably need something like `implicitSubmitLabel` instead of something generic like `label` since it's not a form label, just a button label. But you'd never use it if you have `implicitSubmit=false`, so to keep things in a sane state, I like just keeping it as is.

Open to discuss both if you disagree!

Reply:
ah! I understand now. since `disabled` is indeed different, I do agree that the concise `implicitSubmit` is nice.

Reply:
Yea, I understand. I do think it would be more intuitive in terms of API to split but I don't think it's a big deal if you prefer to leave as is.